### PR TITLE
Add possibility to skip bundler installation

### PR DESCRIPTION
**Pull request by pvlltvk**
Add `rvm1_bundler_install` variable to control bundler installation

### CODE REVIEW

File: tasks/rubies.yml [hunk 1]
```diff
@@ -58,8 +59,18 @@
     dest: '{{ rvm1_symlink_to }}/{{ item }}'
     owner: '{{ root_user }}'
     group: '{{ root_group }}'
-  when: not '--user-install' in rvm1_install_flags
-  with_items: '{{ rvm1_symlink_binaries }}'
+  when: not '--user-install' in rvm1_install_flags and rvm1_bundler_install
+  with_items: '{{ rvm1_symlink_binaries_with_bundler }}'
+
+- name: Symlink ruby related binaries on the system path
```
Comment (pvlltvk) [inline tasks/rubies.yml]: You're right. It's more proper way. I will correct this.

File: tasks/rubies.yml [hunk 2]
```diff
@@ -58,9 +59,19 @@
     dest: '{{ rvm1_symlink_to }}/{{ item }}'
     owner: '{{ root_user }}'
     group: '{{ root_group }}'
-  when: not '--user-install' in rvm1_install_flags
+  when: not '--user-install' in rvm1_install_flags
```
Comment (pkuczynski) [inline tasks/rubies.yml]: Unnecessary white space?

**Comment by pkuczynski** [comment]
@pvlltvk looks good now! Can you only squash your commits before we merge them in?

File: tasks/rubies.yml (hunk 2 above)
Comment (pvlltvk) [inline tasks/rubies.yml]: Yes, I will delete it.

**Comment by pvlltvk** [comment]
@pkuczynski Sure

**Comment by pkuczynski** [comment]
Let me know when ready :)

**Comment by pvlltvk** [comment]
@pkuczynski Done

**Comment by pkuczynski** [comment]
Thanks @pvlltvk!

**Pull request merged**